source/numerics/vector_tools_constraints.inst.in

```diff
@@ -20,74 +20,74 @@ for (deal_II_dimension : DIMENSIONS; deal_II_space_dimension :  SPACE_DIMENSIONS
 {
     namespace VectorTools \{
 
-#if deal_II_dimension == deal_II_space_dimension
 #if deal_II_dimension != 1
+#if deal_II_dimension <= deal_II_space_dimension
     template
     void
-    compute_nonzero_normal_flux_constraints (const DoFHandler<deal_II_dimension> &dof_handler,
+    compute_nonzero_normal_flux_constraints (const DoFHandler<deal_II_dimension,deal_II_space_dimension> &dof_handler,
             const unsigned int     first_vector_component,
             const std::set<types::boundary_id> &boundary_ids,
             FunctionMap<deal_II_dimension>::type &function_map,
```

Comment:
@sairajat Thanks for looking into this. I've just had a quick glance at this, and from the documentation for the [`compute_nonzero_normal_flux_constraints` functions](https://www.dealii.org/8.5.0/doxygen/deal.II/group__constraints.html#ga5b13c2da1b235c86cdbce2270525db99) it would appear that the author intended the template argument for `function_map` to be `FunctionMap<deal_II_space_dimension>`. Would you mind please checking whether this is the case, and if so making the appropriate changes?